src/nvim/api/win_config.c

```diff
@@ -416,6 +416,8 @@ void nvim_win_set_config(Window window, Dict(win_config) *config, Error *err)
     return;
   }
   win_T *parent = config->win == 0 ? curwin : NULL;
+  tabpage_T *parent_tp = parent ? win_find_tabpage(parent) : NULL;
```

Comment:
```suggestion
  tabpage_T *parent_tp = config->win == 0 ? curtab : NULL;
```

Nit: If `parent` is not `NULL` here then it's `curwin`, so it must be in `curtab`; skips a call.
Probably also clearer to make the condition `config->win == 0` here too.